CONTRIBUTING.md

```diff
@@ -21,6 +21,14 @@ For more details follow the [Build and Release documentation](docs/build_release
 
 To get started, refer to the documentation for [setting up a development environment](docs/development.md).
 
+### General contribution guidelines
+
+- A Github issue suggesting the change should be opened prior to a PR.
+- All contributions should be documented, tested and linted. Please verify that all tests and lint checks pass successfully before proposing a change.
+- In order for a pull request to be accepted, the CI (containing unit tests, functional tests and linting) needs to succeed, in addition to approvals from two core contributors.
```

Comment:
"core contributors" vs. "maintainers": is this intentional?

Reply:
No. Is there a difference?